src/components/Dropdowns/AccessibleDropdown.tsx

```diff
@@ -0,0 +1,229 @@
+import { Button, Dropdown, Tooltip } from "antd";
+import useToken from "antd/es/theme/useToken";
+import React, { ReactElement, ReactNode, useEffect, useRef, useState } from "react";
+import styled, { css } from "styled-components";
+
+import { DropdownSVG } from "../../assets";
+import { VisuallyHidden } from "../../styles/utils";
+
+type AccessibleDropdownProps = {
+  /** Text label to include with the dropdown. If null or undefined, hides the label. */
+  label?: string | null;
+
+  /** Contents to be rendered inside the dropdown. */
+  dropdownContent: ReactElement;
+  renderDropdownContent?: ((content: ReactElement) => ReactElement) | null;
+  disabled?: boolean;
+  /** The type of button to render for the dropdown. See Antd's button types:
+   * https://ant.design/components/button#components-button-demo-basic */
+  buttonType?: "primary" | "default" | "outlined" | "dashed" | "text" | "link";
```

Comment:
`outlined` is not a recognized Ant type-- it's similar to `default` but with grey outlines instead of Ant's default purple.

Reply:
*Default:*
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/af4e835f-270b-4e17-a7e7-5dbc4547e54f)

*Outlined:*
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/cfafc6ac-2df7-4a11-81ba-770c0c81ecbb)

Reply:
Yes, fixed!